README.md

```diff
@@ -131,6 +131,7 @@ Use the following environment variables to configure the application:
 | <tt>OWNER_CHECKER_ALLOW_UNOWNED_PATTERNS</tt> | `true`                        | 	Specifies whether CODEOWNERS may have unowned files. For example: <br> <br>  `/infra/oncall-rotator/                    @sre-team` <br>  `/infra/oncall-rotator/oncall-config.yml` <br> <br>  The `/infra/oncall-rotator/oncall-config.yml` file is not owned by anyone.                                                                                                                                                                                       |
 | <tt>OWNER_CHEKER_OWNERS_MUST_BE_TEAMS</tt>    | `false`                       | Specifies whether only teams are allowed as owners of files.                                                                                                                                                                                                                                                                                                                                                                                                    |
 | <tt>NOT_OWNED_CHECKER_SKIP_PATTERNS</tt>      | -                             | The comma-separated list of patterns that should be ignored by `not-owned-checker`. For example, you can specify `*` and as a result, the `*` pattern from the **CODEOWNERS** file will be ignored and files owned by this pattern will be reported as unowned unless a later specific pattern will match that path. It's useful because often we have default owners entry at the begging of the CODOEWNERS file, e.g. `*       @global-owner1 @global-owner2` |
+| <tt>NOT_OWNED_CHECKER_SUBDIRECTORIES</tt>      | -                             | The comma-separated list of subdirectories to check in `not-owned-checker`. When specified, only files in the listed subdirectories will be checked if they do not have specified owners in CODEOWNERS.                                                                                                                                                                                                                                                                                                    |
```

Comment:
It was not clear how to space these descriptions, please let me know if this needs to be changed.

Reply:
it's ok 👍